### PR TITLE
Update the types

### PR DESCRIPTION
**Pull request by nitisht**
Update DirectCSIDrive and DirectCSIVolume types to capture
required info.

### CODE REVIEW

File: pkg/apis/direct.csi.min.io/v1alpha1/types.go [hunk 1]
```diff
@@ -32,37 +32,43 @@ type DirectCSIDrive struct {
...
-	Mountpoint string `json:"mountpoint,omitempty"`
 	// +listType=atomic
 	// +optional
 	MountOptions []string `json:"mountOptions,omitempty"`
 	// +optional
 	DriveStatus DriveStatus `json:"driveStatus,omitempty"`
 	// +optional
 	Topology *csi.Topology `json:"topology,omitempty"`
+	// +optional
+	FormatInfo FormatInfo `json:"formatInfo,omitempty"`
```
Comment (wlan0) [inline pkg/apis/direct.csi.min.io/v1alpha1/types.go:62]: Following our discussion on #direct-csi on slack, let us rename this to RequestedFormat

File: pkg/apis/direct.csi.min.io/v1alpha1/types.go [hunk 2]
```diff
@@ -32,37 +32,43 @@ type DirectCSIDrive struct {
...
+}
+
+type FormatInfo struct {
+	// +optional
+	Forced bool `json:forced,omitempty`
```
Comment (wlan0) [inline pkg/apis/direct.csi.min.io/v1alpha1/types.go:67]: Force bool

**Review by wlan0 — CHANGES_REQUESTED** [review]

**Comment by nitisht** [comment]
@wlan0 addressed the comments, PTAL

**Review by wlan0 — APPROVED** [review]
LGTM

**Comment by wlan0** [comment]
Closing this since the commit was already merged as a part of other PR

**Pull request closed without merging**